run/local-cluster.sh

```diff
@@ -124,35 +123,36 @@ if [[ $DOCKERHOST != "127.0.0.1" ]]; then
 fi
 
 # Get gossip network contents from each node in turn.
-echo -n "Waiting for complete gossip network"
+echo -n "Waiting for complete gossip network of $((NODES*NODES)) peerings: "
 MAX_WAIT=20 # seconds
 for ATTEMPT in $(seq 1 $MAX_WAIT); do
-  echo -n .
-  ALL_FOUND=1
+  FOUND=0
   for i in $(seq 1 $NODES); do
-    GOSSIP=$(curl -s $DOCKERHOST:${HTTP_PORTS[$i]}/_status/gossip)
-    for j in $(seq 1 $NODES); do
-      if [[ $(echo $GOSSIP | grep "node-$j") == "" ]]; then
-        ALL_FOUND=0
-        break
+    FOUND_NAMES=""
+    GOSSIP_URL="$DOCKERHOST:${HTTP_PORTS[$i]}/_status/gossip"
+    GOSSIP=$(curl --noproxy '*' -s $GOSSIP_URL)
+    for j in $(seq 1 $((2*NODES))); do
```

Comment:
node the 2*NODES here, added in to make up for the fact that there's a nodeID gap: node1, node3, node4

I should note that this happens reliably on CircleCI, but not locally on my Debian. Could be a timing issue, but I have no clue.